pkg/ovs/openflow/ofctrl_group.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"fmt"
+
+	"github.com/contiv/libOpenflow/openflow13"
+
+	"github.com/contiv/ofnet/ofctrl"
```

Comment:
This one should be in the same group with the previous one?

Reply:
Updated.